webapp/components/compat-2021.js

```diff
@@ -750,6 +760,30 @@ class Compat2021FeatureChart extends PolymerElement {
       };
     }
 
+    // We draw the chart in two ways, depending on the viewport width. In
+    // 'full' mode the legend is on the right and we limit the chart size to
+    // 700px wide. In 'mobile' mode the legend is on the top and we use all the
+    // space we can get for the chart.
+    //
+    // Google Charts is not responsive, so once drawn the settings are static
```

Comment:
But can one ask it to redo the layout? There's already a resize event listener, and https://stackoverflow.com/questions/41319720/make-google-chart-responsive suggest calling a `drawChart()` which recreates the whole chart. So I guess that's one heavyhanded option.

Reply:
I don't quite follow what you're asking here. The only way to redo the layout is to `drawChart()`, which is exactly what the resize event listener I added in this PR does. Did this comment mislead you in some way? (Note that it is not near the resize listener when reading the code normally; obviously in GitHub code review view the code is collapsed).

Reply:
Yes, I took the comment to mean Google Charts is not responsive (in its layout), and nothing is done to work around that. While it seems like it could be worked around by doing something in a resize listener.